src/System.IO.Pipelines.Networking.Sockets/SocketListener.cs

```diff
@@ -137,22 +137,19 @@ private void OnAccept(SocketAsyncEventArgs e)
             // will cause a stack-dive in the sync (backlog) case
         }
 
-        private async void ExecuteConnection(SocketConnection conn)
+        private async void ExecuteConnection(SocketConnection connection)
```

Comment:
Is why the logging threadpool wrapper in Kestrel; throwing an exception out of a QUWI does the same thing.